addOns/spiderAjax/src/main/java/org/zaproxy/zap/extension/spiderAjax/AjaxSpiderResultsTable.java

```diff
@@ -130,7 +136,7 @@ protected Component doHighlight(Component component, ComponentAdapter adapter) {
             ProcessedCellItem cell = (ProcessedCellItem) adapter.getValue(columnIndex);
 
             boolean processed = cell.getState() == ResourceState.PROCESSED;
```

Comment:
nit: we don't need this variable now, can use this conditional directly in line 147